agent/ecscni/plugin.go

```diff
@@ -240,17 +241,24 @@ func (client *cniClient) createIPAMConfig(cfg *Config) (IPAMConfig, error) {
 		return IPAMConfig{}, err
 	}
 
+	routes := []*cnitypes.Route{
+		{
+			Dst: *dst,
+		},
+	}
+	for _, route := range cfg.AdditionalLocalRoutes {
+		seelog.Debugf("Adding an additional route for %s", route)
```

Comment:
nit: I think there's a `String()` that we should be able to use for this.

Reply:
No, there's no `String()`.